.github/workflows/test-client.yml

```
@@ -318,29 +322,17 @@ jobs:
           echo "CDB_SNAPSHOT_DIGEST=$(${{ steps.command.outputs.mithril_client }} --origin-tag CI cardano-db snapshot list --json | jq -r '.[0].digest')" >> $GITHUB_ENV
 
       - name: Cardano Database Snapshot / download & restore latest
-        if: matrix.extra_args != '--include-ancillary'
         shell: bash
-        run: ${{ steps.command.outputs.mithril_client }} ${{ steps.prepare.outputs.debug_level }} --origin-tag CI cardano-db download $CDB_SNAPSHOT_DIGEST --download-dir /app ${{ matrix.extra_args }}
+        run: ${{ steps.command.outputs.mithril_client }} ${{ steps.prepare.outputs.debug_level }} --origin-tag CI cardano-db download $CDB_SNAPSHOT_DIGEST --download-dir /app/data ${{ matrix.extra_args }}
 
-      - name: Cardano Database Snapshot / download & restore latest and run conversion from InMemory to LMDB
+      - name: Ledger state snapshot conversion from InMemory to LMDB
         if: matrix.extra_args == '--include-ancillary'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         shell: bash
-        run: |
-          docker run --rm \
-            --entrypoint bash \
-            -e NETWORK=$NETWORK \
-            -e GENESIS_VERIFICATION_KEY=$GENESIS_VERIFICATION_KEY \
-            -e ANCILLARY_VERIFICATION_KEY=$ANCILLARY_VERIFICATION_KEY \
-            -e AGGREGATOR_ENDPOINT=$AGGREGATOR_ENDPOINT \
-            -e GITHUB_TOKEN=$GITHUB_TOKEN \
-            --name='mithril-client' \
-            ghcr.io/input-output-hk/mithril-client:$MITHRIL_IMAGE_ID \
-            -c "
-              /app/bin/mithril-client --origin-tag CI cardano-db download $CDB_SNAPSHOT_DIGEST --download-dir /app --include-ancillary &&
-              /app/bin/mithril-client --unstable tools utxo-hd snapshot-converter --db-directory /app/db --cardano-node-version latest --utxo-hd-flavor LMDB --cardano-network \$NETWORK --commit
-            "
+        run: ${{ steps.command.outputs.mithril_client }} ${{ steps.prepare.outputs.debug_level }}  --unstable tools utxo-hd snapshot-converter --db-directory /app/data/db --cardano-node-version latest --utxo-hd-flavor LMDB --cardano-network $NETWORK --commit
+
+      - name: Remove downloaded artifacts to free up disk space
+        shell: bash
+        run: rm --force /app/data/db/immutable/*.{chunk,primary,secondary}
```

Since the removal is executed on the host computer I don't think that the provided path is correct
```suggestion
        run: rm --force $PWD/data/db/immutable/*.{chunk,primary,secondary}
```

---

You're absolutely right, thanks.